compiler/lib-wasm/gc_target.ml

```diff
@@ -989,22 +982,18 @@ module Constant = struct
     | NativeString s ->
         let s =
           match s with
-          | Utf (Utf8 s) -> str_js_utf8 s
-          | Byte s -> str_js_byte s
+          | Utf (Utf8 s) -> s
+          | Byte s -> byte_string s
         in
-        let* i = register_string s in
         let* x =
-          let* name = unit_name in
           register_import
-            ~import_module:
-              (match name with
-              | None -> "strings"
-              | Some name -> name ^ ".strings")
-            ~name:(string_of_int i)
-            (Global { mut = false; typ = Ref { nullable = false; typ = Any } })
+            ~import_module:""
```

Comment:
It would look less magic if we use an import_module that is not the empty string. What do you think ?

Reply:
The module name is repeated for each string, so it is better to keep it short. On the other hand, we probably do not have many JavaScript strings in practice.